src/api/manager.py

```diff
@@ -132,7 +132,7 @@ def save(self, data):
         data = self.clean_data(data)
         data = self.add_created(data)
         result = self.db.insert_one(self.convert_data(data))
-        return {"_id": str(result.inserted_id)}
+        return {"_id": str(result.inserted_id), "name":data["name"]}
```

Comment:
Shouldn't be returning "name" here because then every schema has to have a name field. Whatever is calling this method will already have the name so it doesn't need to be returned back.